test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -759,6 +789,170 @@ else if (cluster instanceof Cluster)
         initialized = true;
     }
 
+    private void startJmx() throws InterruptedException
```

Comment:
this is more of a nit than an actual comment, but do you think it would be possible to move JMX-related initializers to their own class? I know we already got the jmx util class in .15, but we could also have one cassandra-locally. Most other initializations live in messaging, gossip, commit log, and other subsystems. I understand this is not necessarily matching a pattern here, but maybe we we could have a class under /impl/

Reply:
Done - it really helps clean things up as well, so thanks for the suggestion.